app/tests/src/tokenBundle.js

```diff
@@ -0,0 +1,110 @@
+import assert from 'assert'
```

Comment:
This PR seems to be based on the one fixing the assets ordering, but this file is named tokenFormatter.js there: https://github.com/vacuumlabs/adalite/pull/922/files#diff-455d601662bff42e0cc51f668eab06af1897ae39c9330ba6ec1f3928119d231f - so make sure to not accdentally introduce a duplicate test when merging/rebasing. I like the tokenBundle.js name more if I had to pick one